packages/analytics/metric-cards/src/components/display/MetricsCard.vue

```diff
@@ -11,7 +11,11 @@
         :color="KUI_COLOR_TEXT_NEUTRAL"
         :size="KUI_ICON_SIZE_30"
       />
-      <component :is="titleTag">
+      <span v-if="!titleTag">{{ title }}</span>
```

Comment:
Instead of the `v-if`, can we just default the `titleTag` property to `span` (pending discussion)?